File: C11/백준/Bronze/11720. 숫자의 합/숫자의 합.c
#include <stdio.h>

int main(void){
	
	int N;
	scanf("%d", &N);
	
	char numbers[N + 1];
	scanf("%s", numbers);
	
	int sum = 0;
	
	for(int i = 0; i < N; i++){
		int number = numbers[i] - '0';
		sum += number;
	}
	
	printf("%d", sum);
	
	return 0;
}

File: Java/백준/Bronze/11720. 숫자의 합/숫자의 합.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {

        Scanner s = new Scanner(System.in);

        int N = s.nextInt();
        String numbers = s.next();
        int sum = 0;

        for (int i = 0; i < N; i++) {
            int number = numbers.charAt(i) - '0';
            sum += number;
        }

        System.out.print(sum);

    }
}